src/airtable/project_summary.js

```diff
@@ -0,0 +1,135 @@
+// @format
+require("dotenv").config();
+const process = require("process");
+const Airtable = require("airtable");
+
+const { AIRTABLE_API_KEY, AIRTABLE_BASEID } = process.env;
+const base = new Airtable({ apiKey: AIRTABLE_API_KEY }).base(AIRTABLE_BASEID);
+
+const fields = [
+	"Project Name",
+	"OCEAN Granted",
+	"Voted Yes",
+	"Voted No",
+	"Proposal Standing"
+];
+
+const maxRecordAmount = 10;
+
+const chunk = projects => {
+	const chunks = [];
+	for (i = 0, j = projects.length; i < j; i += maxRecordAmount) {
+		chunks.push(projects.slice(i, i + maxRecordAmount));
+	}
+	return chunks;
+};
+
+const populate = async projects => {
+	return new Promise((resolve, reject) => {
+		base("Project Summary").create(projects, (err, records) => {
```

Comment:
I've noticed that `create` simply appends all records to the end of the table, so I'm wondering if we have to decide between `create` and whatever function "update" is. An alternative could be to simply delete the table upon each renewed population.

E.g. I could image the script running into problems where we updated based on the project name as the unique identifier as then e.g. it'd try to update a non-existed record. In that case, deleting everything and recomputing would be better.

Thoughts @idiom-bytes? How is this done with the other tables?